web/index.css

```diff
@@ -385,6 +385,12 @@ div.toggle {
   background: #3f3f3f;
 }
 
+.sharing-dialog {
```

Comment:
I think this needs to be merged with the latest. The sharing dialog isn't positioned any differently than the other ones.